### ISSUE TITLE
Error message routeCheck failed when running vrf test

### ISSUE THREAD

**Issue by AndriiLozovyi** [Azure/sonic-utilities#1229]
**Description:** 
Error message is seen due to is_local() expecting to receive an Ipv4 or Ipv6 address but receives an ROUTE_TABLE keys from APPL_DB which look like this **Vrf2:fc00::7c**

```
def is_local(ip):
    t = ipaddress.ip_address(ip.split("/")[0].decode('utf-8'))
    return t.is_link_local
```

**Steps to reproduce:** 
1) Deploy topology t0 
2) Deploy Minigraph
3) Run test_vrf.py::TestVrfLoopbackIntf

**Errors in Syslog:**
```
cab18-2-dut ERR monit[557]: 'routeCheck' status failed (1) -- Traceback (most recent call last):#012  File "/usr/local/bin/route_check.py", line 265, in <module>#012    main(sys.argv[1:])#012  File "/usr/local/bin/route_check.py", line 256, in main#012    ret = check_routes()#012  File "/usr/local/bin/route_check.py", line 200, in check_routes#012    rt_appl = get_routes()#012  File "/usr/local/bin/route_check.py", line 120, in get_routes#012    if not is_local(k):#012  File "/usr/local/bin/route_check.py", line 67, in is_local#012    t = ipaddress.ip_address(ip.split("/")[0].decode('utf-8'))#012  File "/usr/local/lib/python2.7/dist-packages/ipaddress.py", line 168, in ip_address#012    address)#012ValueError: u'Vrf2:fc00::7c' does not appear to be an IPv4 or IPv6 address
```

**Describe the results you expected**

**Additional information you deem important (e.g. issue happens only occasionally)**

**Output of `show version`**
```
SONiC Software Version: SONiC.master.28-dirty-20201105.040821
Distribution: Debian 10.6
Kernel: 4.19.0-9-2-amd64
Build commit: ce6286eb
Build date: Thu Nov  5 14:36:39 UTC 2020
Built by: johnar@jenkins-worker-7

Platform: x86_64-accton_wedge100bf_32x-r0
HwSKU: montara
ASIC: barefoot
Serial Number: AH47035964
```

**Comment by anshuv-mfst** [comment]
Issue Triage 4/14: 
@renukamanavalan could you please take a look, thanks.

**Comment by renukamanavalan** [comment]
Fix checked in[ PR #1565](https://github.com/Azure/sonic-utilities/pull/1565)
